static/tipuesearch/tipuesearch_content.js

```diff
@@ -0,0 +1,17 @@
+
```

Comment:
this should be empty in the theme, this is a generated file when you run pelican command, not even sure if should live here in this folder.

Reply:
Removed as per latest commit, thank you